### ISSUE TITLE
✏️ Remove Definition of Ready

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#12241]
## ✏️ Documentation

The **Definition of Ready** is [vague](https://github.com/GCTC-NTGC/gc-digital-talent/discussions/7227) and, as far as I know, is not used or referenced by anyone on the team.

## 🙋‍♀️ Proposed Solution

Remove file `documentation/definition-of-ready.md`.

## ✅ Acceptance Criteria

- [ ] `documentation/definition-of-ready.md` no longer exists in codebase

**Comment by brindasasi** [comment]
we should be having DoR and DoD. If it is not working , let's discuss on how we can improve. May be on next retro ?

**Comment by tristan-orourke** [comment]
I agree with Brinda. We should discuss this in retro, not scrap the concept entirely.
